core/src/main/java/org/opensearch/sql/planner/physical/ProjectOperator.java

```diff
@@ -27,10 +35,13 @@
 @EqualsAndHashCode
 @RequiredArgsConstructor
 public class ProjectOperator extends PhysicalPlan {
+  private static final Logger log = LogManager.getLogger(ProjectOperator.class);
   @Getter
   private final PhysicalPlan input;
   @Getter
   private final List<NamedExpression> projectList;
+  @Getter
+  private final Map<String, ParseExpression> parseExpressionMap;
```

Comment:
Could we reuse NamedExpression instead of Map<String, ParseExpression>

Reply:
updated to use another `List<NamedExpression>` which store the parseExpressions